api/role/schema.py

```diff
@@ -17,6 +18,9 @@ class Arguments:
     role = graphene.Field(Role)
 
     def mutate(self, info, **kwargs):
+        new_role = RoleModel.query.filter_by(role=kwargs.get('role')).first()
```

Comment:
Use sqlalchemy exceptions to catch more errors, e.g db conflict, db connection, validation errors etc, this can be in a try except block or wrapped in a context manager, using a context manger will help us solve for duplication as well